### PR TITLE
Feature/admin user profiles (wip)

### PR DESCRIPTION
**Pull request by kazoompa**

### CODE REVIEW

File: opal-ui/src/stores/profiles.ts [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+  }
+
+  async function deleteProfiles(profiles: SubjectProfileDto[]) {
+    const principals = profiles.map((profile) => profile.principal);
+    return api.delete('/system/subject-profiles',{
+      params: {
+          p: principals
+      },
+      paramsSerializer: params => {
+        return qs.stringify(params, { arrayFormat: 'repeat' });
```
Comment (ymarcon) [inline opal-ui/src/stores/profiles.ts:39]: You do not need the `qs` dependency:

```
paramsSerializer: {
    indexes: null, // no brackets at all
}
```

**Review by ymarcon — CHANGES_REQUESTED** [review]

File: opal-ui/src/stores/profiles.ts (hunk 1 above)
Comment (kazoompa) [reply opal-ui/src/stores/profiles.ts]: Done

File: opal-ui/package-lock.json [hunk 2]
```diff
@@ -1435,6 +1435,21 @@
       "integrity": "sha512-Tpp60P6IUJDTuOq/5Z8cdskzJujfwqfOTkrwIwj7IRISpnkJnT6SyJ4PCPnGMoFjC9ddhal5KVIYtAt97ix05A==",
       "dev": true
     },
+    "node_modules/body-parser/node_modules/qs": {
```
Comment (ymarcon) [inline opal-ui/package-lock.json:1438]: Still in the package lock.

**Review by ymarcon — CHANGES_REQUESTED** [review]

File: opal-ui/package-lock.json (hunk 2 above)
Comment (kazoompa) [reply opal-ui/package-lock.json:1438]: This dep has always been there: `git diff 07003991723de735480f8cdb847750ac6c6d6e33 -- package-lock.json`

**Review by ymarcon — APPROVED** [review]

**Pull request merged**